src/igvfd/schemas/analysis_step_version.json

```diff
@@ -0,0 +1,79 @@
+{
+    "title": "Analysis Step Version",
+    "description": "A step version in a computational analysis workflow.",
+    "$id": "/profiles/analysis_step_version.json",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "required": [
+        "lab",
+        "award",
+        "analysis_step",
+        "software_versions"
+    ],
+    "identifyingProperties": [
+        "uuid",
+        "aliases",
+        "name"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/basic_item"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/standard_status"
+        },
+        {
+            "$ref": "mixins.json#/release_timestamp"
+        }
+    ],
+    "dependentSchemas": {
+        "$merge": [
+            "shared_dependencies.json#/standard_status"
+        ]
+    },
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "analysis_step": {
+            "title": "Analysis Step",
+            "description": "The analysis step in which this version belongs to.",
```

Comment:
This sentence is confusing to me, maybe "The analysis step which this version belongs to"?

Reply:
done